src/display-api/display-api.service.ts

```diff
@@ -126,4 +133,155 @@ export class DisplayApiService {
       );
     }
   }
+
+  /**
+   * Fetches all screens.
+   */
+  async fetchScreens(): Promise<ScreenScreenJsonld[]> {
+    try {
+      const config = await this.getAuthenticatedConfig();
+      const screensApi = new ScreensApi(config);
+      const screens: ScreenScreenJsonld[] = [];
+
+      let page = 1;
+      const itemsPerPage = "24";
```

Comment:
This seems a bit random, but reading on I understand how this is actually ok. Maybe throw in a comment saying that we set a page-size that should be safe but as we intend to iterate all pages to get all responses the actual value of this constant should not be that important which is why we hardcode it?